src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -515,6 +519,31 @@ public void SetDatabaseChangeVector(DocumentsOperationContext context, string ch
             }
         }
 
+        public static string GetFullDatabaseChangeVector(DocumentsOperationContext context)
+        {
+            var tx = context.Transaction.InnerTransaction;
+            var tree = tx.ReadTree(GlobalTreeSlice);
+            var val = tree.Read(GlobalFullChangeVectorSlice);
+            if (val == null)
+            {
+                return string.Empty;
```

Comment:
Let's return the normal database change vector here.
Upgrading an existing server will have null as full database change vector..

Reply:
ok